renovate.json

```diff
@@ -11,7 +11,7 @@
     },
     {
       "matchManagers": ["gradle"],
-      "minimumReleaseAge": 3
+      "minimumReleaseAge": "3 days"
```

Comment:
Should fix https://github.com/ministryofjustice/hmpps-adjudications-insights-api/issues/33